src/defines/medals.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "name": "He4rt",
+    "medal_role_id": "1107790862869614684",

```

Comment:
Is this correct He4rt ID Role? Snowflake are so long, seems to me to be a role created recently.

Reply:
Yes, i created recently